src/js/views/right_panel_view.js

```diff
@@ -548,6 +549,79 @@
             return false;
         },
 
+        initialize_sortable: function() {
+            var self = this;
+            var containers = self.$el.find('.labels-container');
+
+            // Check if there are any elements
+            if (containers.length === 0) {
+                return;
+            }
+
+            // Destroy existing Sortable instances if any
+            if (self.sortables) {
```

Comment:
PR looks great and works well!

The only question I have is about the `destroy()` here... 
If I create a sortable, then select a different panel in the figure then this `SelectedPanelsLabelsView` gets lost without destroying the selectable(s) that were created.
I don't know if that really matters - probably not. Does it hurt if we don't clear destroy them? E.g. use up memory etc??
If we do want to make sure they all get destroyed, I wonder if a more global list of sortables (instead of just `this` component) or possibly try to handle the removal of the component and destroy them there. E.g. see https://github.com/ome/omero-figure/blob/64566080f813fb5d70a77c3cd1a141bf30eaddba/src/js/views/right_panel_view.js#L756 (which was originally created to handle the destruction of jQuery sliders - now I don't use jQuery sliders, so it does less now).

Reply:
I followed the clear example and now destroy the sortables when the panel is deselected. Thanks for spotting that